assets/js/hooks/TabSelector.ts

```diff
@@ -55,7 +56,7 @@ export default {
     });
 
     const config = { childList: true, subtree: true };
-    observer.observe(document.body, config);
+    this.observer.observe(document.body, config);
```

Comment:
As discussed earlier today, this reacts to ever since Dom change - I did try replacing it with a `this.hashChanged(this.getHash() || this.defaultHash);` - i.e. set it when the component is mounted and listen to `hashchange` on the window and it _appeared_ to work; could you test that out because the Dom wide observer is doing doo much work.

Reply:
Yes, I was attempting to fix that behavior by at least making sure that we clear the observer when destroying the hook. But now I have reverted that. So it's no longer part of this PR and when I re base I will get your fix in. Thanks